crates/apub/src/objects/post.rs

```diff
@@ -220,14 +231,16 @@ impl Object for ApubPost {
       None
     };
 
+    let url_blocklist = get_url_blocklist(context).await?;
+
     if let Some(url) = &url {
+      is_url_blocked(url, &url_blocklist)?;
```

Comment:
This isn't anything to do with the issue, I just happened to notice this was missing when looking at the code.

Reply:
Isn't URL blocklist enforced only via API intentionally currently?

Simply dropping these posts when receiving them is a bad solution, as the posts will stay up on the originating instance and community mods will never see them to be able to remove them. For example, if sh.itjust.works uses the URL blocklist feature to ban tinyurl.com and a lemm.ee user posts a link to tinyurl.com to the SJW community the post will be visible for all lemm.ee users but the community moderators will never see the post (unless they're located on lemm.ee).

There are already several other cases that can lead to behavior similar to this, but we should probably not add more to that list.

Reply:
There might be a middle ground similar to how the slur filter works by censoring URLs instead.

Reply:
I'm open to alternatives, but currently we do drop posts if hey have a [blocked URL in the post body](https://github.com/LemmyNet/lemmy/blob/128e78f7c22279fec27d3213a05b440cb1f8db59/crates/apub/src/objects/post.rs#L233). This just makes the behaviour consistent.